### PR TITLE
M sc1 61 19

### PR DESCRIPTION
**Pull request by sergey-mario**

### CODE REVIEW

File: smaretski/03_HTML/task_01_forms.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+		</th>
+		<th align="left" valign="top">
+		<b>Валидация данных HTML5:</b>
+		<p><form id="my_form_8" name="html5" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Строка поиска</legend>
+		<input type="text" lang="ru" pattern="[А-Я][а-Я][А-я][а-я]+" placeholder="Только русские буквы">
+		<br>
+		<br>
+		<input type="tel" name="tel_numb" pattern="\+375-(29|44|33|25)-[0-9]{7}" placeholder="+375-29-1234567" maxlength="15">
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: Wrong pattern

File: smaretski/03_HTML/task_01_forms.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+		<br>
+		<input type="text" name="name_01" pattern="[0-9]{4}">
+		<input type="submit" name="download" value="загрузить">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		<th align="left" valign="top">
+		<b>Валидация данных HTML5:</b>
+		<p><form id="my_form_8" name="html5" method="get" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: remove p

Comment (pluhin) [reply smaretski/03_HTML/task_01_forms.html]: do not put tag form inside p.
p - for human text should be paragraph with text

File: smaretski/03_HTML/task_01_forms.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+		<th align="left" valign="top">
+		<b>Загрузка файла на сервер</b>
+		<p><form id="my_form_7" name="upload" method="post" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Upload</legend>
+		Файл:
+		<br>
+		<input type="file" name="file_field" dir="ltr">
+		<br>
+		Срок хранения (года):
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: label here

File: smaretski/03_HTML/task_01_forms.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Загрузка файла на сервер</b>
+		<p><form id="my_form_7" name="upload" method="post" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Upload</legend>
+		Файл:
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: label here

File: smaretski/03_HTML/task_01_forms.html [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		<th align="left" valign="top">
+		<b>Выпадающий и мульти списки</b>
+		<p><form id="my_form_6" name="list_city" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Города РБ</legend>
+		<select name="list" id="list_blr">
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: duplicate name with line 117

File: smaretski/03_HTML/task_01_forms.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+		<tr>
+		<th align="left" valign="top">
+		<b>Опрос (голосование):</b>
+		<p><form id="my_form_5" name="feedback" method="get|post" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Оставьте Ваш отзыв</legend>
+		<input type="text" name="login" required><span>*</span>Имя
+		<br>
+		<br>
+		<input type="email" name="email" required><span>*</span>e-mail
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Опрос (голосование):</b>
+		<p><form id="my_form_5" name="feedback" method="get|post" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Оставьте Ваш отзыв</legend>
+		<input type="text" name="login" required><span>*</span>Имя
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+		<input type="hidden" name="data_voite" value="2013.12.31">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Опрос (голосование):</b>
+		<p><form id="my_form_5" name="feedback" method="get|post" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: wrong method

File: smaretski/03_HTML/task_01_forms.html [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+		</form>
+		</p>
+		</th>
+		</th>
+		<th align="left" valign="top">
+		<b>Опрос (голосование):</b>
+		<p><form id="my_form_4" name="vote" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Как вам?</legend>
+		<input type="radio" name="radio_vote" value="vote_1" id="checkbox_vote_1" checked>ничего
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html:71]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+		<br>
+		<input type="radio" name="radio_vote" value="vote_2" id="checkbox_vote_2" checked>неплохо
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html:75]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+		<br>
+		<input type="radio" name="radio_vote" value="vote_3" id="checkbox_vote_3" checked>хорошо
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html:79]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+		<br>
+		<input type="radio" name="radio_vote" value="vote_4" id="checkbox_vote_4" checked>супер
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html:82]: label

File: smaretski/03_HTML/task_01_forms.html [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Простая форма авторизации:</b>
+		<p><form id="my_form_3" name="author" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Авторизируйтесь</legend>
+		<input type="text" name="login" required> <span>*</span>Логин
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: missing autofocus

File: smaretski/03_HTML/task_01_forms.html [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+		</label>
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Простая форма авторизации:</b>
+		<p><form id="my_form_3" name="author" method="get" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline smaretski/03_HTML/task_01_forms.html]: you send password by method get. it is a joke?

File: smaretski/02_HTML/task_03_link_img.html [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+def DoTwo(num1,num2):
+        print '%d + %d = %d ' % (num1,num2,num1+num2)
+        print '%d * %d = %d ' % (num1,num2,num1*num2)
+        print '%d - %d = %d ' % (num1,num2,num1-num2)
+        print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)</pre>
+<p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет чтолибо поменять в функции, мы сможем сделать это, не создавая проблем для нашей программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p>
+        <center><table width="100%" border="0"</table></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_03_link_img.html:63]: could you please remove tag center here

File: smaretski/02_HTML/task_02_table.html [hunk 16]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>МОЁ ДР</TITLE> 
+    </HEAD>
+    <BODY> 
+        <center><H1>Месяц моего дня рождения</H1></center>
+		<center><table width="200" border="2"></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_02_table.html:9]: could you please remove tag center here

File: smaretski/02_HTML/task_02_table.html [hunk 17]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>МОЁ ДР</TITLE> 
+    </HEAD>
+    <BODY> 
+        <center><H1>Месяц моего дня рождения</H1></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_02_table.html:8]: could you please remove tag center here

File: smaretski/02_HTML/task_02_link_img.html [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+        <p>Ещё один пример списков — категории в поваренной книге. Например…</p>
+        <pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+        <p>Таким образом, в <b>categories[0]</b> хранится <i>'Main dish'</i>, а в <b>categories[4]</b> — <i>'Cookies'</i>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+        <h2 id="comments">Комментарии</h2>
+        <p><a href="#content">К содержанию</a></p>
+        <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+        <pre>&#35</pre>
+        <p>Это комментарий</p>
+        <p>Комментарии можно вставлять в любое место в строке, но помните, что Python проигнорирует всё, что идёт после '#'.</p>
+        <center><table width="100%" border="0"</table></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_02_link_img.html:40]: could you please remove tag center here

**Review by pluhin — CHANGES_REQUESTED** [review]
Do not use tag center, find another  solution to center content

File: smaretski/03_HTML/task_01_forms.html (hunk 1 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Do you mean string #158 or #161 ? Or both of them?

File: smaretski/03_HTML/task_01_forms.html (hunk 2 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Fixed. I removed tag <p> in file

File: smaretski/03_HTML/task_01_forms.html (hunk 14 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Fixed. Replaced by method post

File: smaretski/03_HTML/task_01_forms.html (hunk 13 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Fixed. Added autofocus

File: smaretski/03_HTML/task_01_forms.html (hunk 9 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html:71]: Added label. My bad in all file :)

File: smaretski/03_HTML/task_01_forms.html (hunk 10 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html:75]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 11 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html:79]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 12 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html:82]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 8 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Method changed by "post"

File: smaretski/03_HTML/task_01_forms.html (hunk 7 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 6 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 5 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Rename and use different name

File: smaretski/03_HTML/task_01_forms.html (hunk 4 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 3 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Added label

File: smaretski/03_HTML/task_01_forms.html (hunk 1 above)
Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: I can't understand what is wrong. I checked different phonenumbers and it always work.

Comment (pluhin) [reply smaretski/03_HTML/task_01_forms.html]: There is +XXX-XX-XXXXXXX pattern in hometask

Comment (sergey-mario) [reply smaretski/03_HTML/task_01_forms.html]: Fixed. Please, check this.

**Pull request merged**